components/sandbox/toolchain-member-operator/base/rbac/appstudio-pipelines-runner.yaml

```diff
@@ -33,6 +33,7 @@ rules:
       - watch
     apiGroups:
       - toolchain.dev.openshift.com
+      - eaas.konflux-ci.dev
```

Comment:
We don't want to set permissions for kind `spacerequests` in group `eaas.konflux-ci.dev`. That resource doesn't exist.

Instead we want to set permission for kind `namespaces` in group `eaas.konflux-ci.dev`. A new rule is required to do this.